src/Entity/DatasetSubmission.php

```diff
@@ -2662,4 +2672,26 @@ public function removeDatasetLink(DatasetLink $datasetLink): self
 
         return $this;
     }
+
+    /**
+     * Getter for remotely hosted url.
+     *
+     * @return string
+     */
+    public function getRemotelyHostedUrl(): string
```

Comment:
Got this error:
`Return value of App\Entity\DatasetSubmission::getRemotelyHostedUrl() must be of the type string, null returned`